doc/funcref.sgml

```diff
@@ -2587,6 +2567,40 @@ ldiv
 </quote>
 
 
+<sect1>doesclrscrafterexit<label id="doesclrscrafterexit"><p>
+
+<quote>
+<descrip>
+<tag/Function/Determines whether the screen is going to be cleared after program exit.
+<tag/Header/<tt/<ref id="atari.h" name="atari.h">, <ref id="apple2.h" name="apple2.h">/
+<tag/Declaration/<tt/unsigned char doesclrscrafterexit (void);/
+<tag/Description/The function returns 0 if the screen won't be cleared immediately after
+program termination. It returns 1 if it will.
```

Comment:
The Apple II variant doesn't return 1 but some non-0 value. The usual way to phrase that in cc65 is something like "The function returns a value of zero if <...>. It returns non zero otherwise."